GUIMainWindow.py

```diff
@@ -85,23 +89,31 @@ def drawMenuBar(self):
         self.menuObjects["Delete Relationship"] = menuRelationship.addAction("Delete Relationship")
         self.menuObjects["Change Relationship"] = menuRelationship.addAction("Change Relationship")
 
-    #def paintEvent(self, event):
-        #paint = QPainter()
-        #paint.begin(self)
-        #self.drawLines(paint)
-        #paint.end()
+    def paintEvent(self, event):
+        paint = QPainter()
+        paint.begin(self)
+        paint.fillRect(0, 0, 1000, 800, QBrush(Qt.white))
+        self.drawLines(paint)
+        paint.end()
 
-    #def drawLines(self, paint):
-        #penSolid = QPen(Qt.black, 2)
-        #paint.setPen(penSolid)
+
+    def drawLines(self, paint):
+        penSolid = QPen(Qt.black, 2)
+        paint.setPen(penSolid)
         #paint.drawLine(400, 100, 200, 100)
+        for key, x in self.RelationshipCoordiantes.items():
+            paint.drawLine(x[0], x[1], x[2], x[3])
+
+
+    def addRelationshipLine(self, firstClassName, secondClassName, x1, y1, x2, y2):
+        self.RelationshipCoordiantes[(firstClassName, secondClassName)] = [x1, y1, x2, y2]
+
+    def deleteRelationshipLine(self, firstClassName, secondClassName):
+        del self.RelationshipCoordiantes[(firstClassName, secondClassName)]
+
+
 
-        #penDashed = QPen(Qt.black, 2)
-        #penDashed.setStyle(Qt.DashLine)
-        #paint.setPen(penDashed)
-        #paint.drawLine(400, 400, 200, 400)
 
-    # ----------- Anything below this line can be ignored until sprint 3 ----------- #
```

Comment:
Going to bring this line back for my own reference, if we decide we want it out for this sprint I can take it out again